semanticdb/metap/src/main/scala/scala/meta/internal/metap/Settings.scala

```diff
@@ -0,0 +1,31 @@
+package scala.meta.internal.metap
+
+final case class Settings(
+    paths: List[String] = Nil,
+    pretty: Boolean = true,
```

Comment:
> every boolean option is an enum screaming to get out
> -- Ólafur Páll Geirsson

```scala
sealed trait OutputFormat
object OutputFormat {
  case object Pretty extends OutputFormat
  case object Proto extends OutputFormat
}
```

Reply:
Also helps follow the "Make invalid states unrepresentable" principle

Reply:
👍